solarwinds_apm/apm_logging.py

```diff
@@ -38,10 +38,45 @@
 import os
 
 
+class ApmLoggingType:
+    """Mapping of supported solarwinds_apm library log types"""
+
+    log_types = {
+        "STDERR": 0,
+        "FILE": 2,
+        "DISABLED": 4,
+    }
```

Comment:
I've not included stdout nor null because there currently are no cases where APM Python would need to set them. But if too strict I can add them.